### PR TITLE
Templated iteration to layouts

### PR DESCRIPTION
**Pull request by DZ-FSDev**
Hi Sam! I noticed the layouts are very SVG prototype friendly so I templated some changes. The idea is that you can paste your internal CSS and HTML into the SVG file and with a minor modification, make a light-weight embeddable prototype.

### CODE REVIEW

File: ViewPortFluidDesign/image/Layout01.svg [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	height: 100%;
+}
+
+body {background: #c8e6c9; height:100%;}
+#wrap {width: 100%; height: 100%; margin: 0 auto; font-size: 4vh; color:#fff; text-align: center; text-transform: uppercase;}
+#header {width: 100%; height: 8%; background: #2e7d32;}
+#nav {width: 100%; height: 8%; background: #388e3c;}
+#side {float: left; width: 30%; height: 70%; background: #43a047;}
+#contents {float: left; width: 70%; height: 70%; background: #4caf50;}
+#footer {float: left; width: 100%; height: 8%; background: #66bb6a;}
```
Comment (DZ-FSDev) [inline ViewPortFluidDesign/image/Layout01.svg:33]: Styles from the HTML

File: ViewPortFluidDesign/image/Layout01.svg [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+<div class="container" xmlns="http://www.w3.org/1999/xhtml">
+
+<body>
+<div id="wrap">
+<div id="header">header</div>
+<div id="nav">nav</div>
+<div id="side">side</div>
+<div id="contents">contents</div>
+<div id="footer">footer</div></div>
+</body>
```
Comment (DZ-FSDev) [inline ViewPortFluidDesign/image/Layout01.svg:54]: Code from HTML

File: ViewPortFluidDesign/layout01.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+<meta name="viewport" content="width=device-width, initial-scale=1.0 maximum-scale=1.0; user-scalable = no;">
+<title>Standard Web Layout01</title>
+<style>
+	body {background: #c8e6c9; height:100vh;}
+	#wrap {width: 100%; height: 100%; margin: 0 auto; font-size: 4vh; color:#fff; text-align: center; text-transform: uppercase;}
+	#header {width: 100%; height: 8%; background: #2e7d32;}
+	#nav {width: 100%; height: 8%; background: #388e3c;}
+	#side {float: left; width: 30%; height: 70%; background: #43a047;}
+	#contents {float: left; width: 70%; height: 70%; background: #4caf50;}
+	#footer {float: left; width: 100%; height: 8%; background: #66bb6a;}
```
Comment (DZ-FSDev) [inline ViewPortFluidDesign/layout01.html:15]: Styles to put into SVG

File: ViewPortFluidDesign/layout01.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+</head>
+<body>
+<div id="wrap">
+	<div id="header">header</div>
+	<div id="nav">nav</div>
+	<div id="side">side</div>
+	<div id="contents">contents</div>
+	<div id="footer">footer</div>
+</div>
+</body>
```
Comment (DZ-FSDev) [inline ViewPortFluidDesign/layout01.html:26]: Code to put into SVG

**Review by DZ-FSDev — COMMENTED** [review]
Workflow for #1

File: ViewPortFluidDesign/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+
+<!-- CONTENTS -->
+## Contents
+
+### Basic Web Design (Layout 01, 02, 03, 04)
+
+| Layout 01                 | Layout 02                |
+|---------------------|-----------------------|
+| ![](./image/Layout01.svg) | ![](./ViewPortFluidDesign/image/Layout1.svg) |
```
Comment (DevSamLee) [inline ViewPortFluidDesign/README.md:17]: I like the way you've changed. (jpg to svg)

**Review by DevSamLee — APPROVED** [review]
#1  Looks good!

**Pull request merged**